tsconfig.base.json

```diff
@@ -17,7 +17,8 @@
     "paths": {
       "@zupit-it/ng-auth": ["packages/ng-auth/src/index.ts"],
       "@zupit-it/ng-form": ["packages/ng-form/src/index.ts"],
-      "@zupit-it/tools": ["packages/tools/src/index.ts"]
+      "@zupit-it/tools": ["packages/tools/src/index.ts"],
+      "@zupit-it/zu-icon": ["packages/zu-icon/src/index.ts"]
```

Comment:
I think there should be an update of the name and the path

Reply:
my bad, fixed